unix/src/machine_stats/__init__.py

```diff
@@ -154,6 +131,34 @@ def v2_runner_on_ok(self, result):
             }
         )
 
+    def v2_playbook_on_stats(self, stats):
+        if self._total_results is not None:
+            print(json.dumps(self._total_results, indent=4, sort_keys=True))
+
+        self._display.banner("MACHINE STATS RECAP")
+
+        hosts = sorted(stats.processed.keys())
+        for h in hosts:
+            t = stats.summarize(h)
+
+            self._display.display(
+                u"%s : %s %s %s %s %s %s %s"
+                % (
+                    hostcolor(h, t),
+                    colorize(u"ok", t["ok"], C.COLOR_OK),
```

Comment:
nice,  😍 . 
Would you be able to share a quick screenshot of how it looks?  (PR description or other means)